documentation/sphinx/source/api-common.rst.inc

```diff
@@ -588,3 +588,48 @@
 .. |locality-get-addresses-for-key-blurb| replace::
 
     Returns a list of public network addresses as strings, one for each of the storage servers responsible for storing ``key`` and its associated value.
+
+.. |option-knob| replace::
+   
+   Sets internal tuning or debugging knobs.
```

Comment:
This one is complicated enough to use that it might be good to explain what you need to pass in as the value.

Reply:
I added link to the knobs. I think we need to have separate documentation for useful knobs.